pinot-query-runtime/src/test/resources/queries/FromExpressions.json

```diff
@@ -136,6 +136,11 @@
         "description": "sub-query to semi-join syntax with star results, using IN clause",
         "sql": "SELECT * FROM {tbl1} WHERE num IN (SELECT num FROM {tbl2})"
       },
+      {
+        "psql": "7.2.1.3",
+        "description": "sub-query to multiple anti semi-join queries with star results, using NOT IN clause",
+        "sql": "SELECT * FROM {tbl1} WHERE (num > -10 and num < 10) AND (name NOT IN (SELECT val FROM {tbl2} WHERE num = 3)) AND (name NOT IN (SELECT val from {tbl2} WHERE num = 4))"
```

Comment:
could you add additional test for the range predicate also inside the NOT-IN subquery?

Reply:
Done. Can you check again?